.github/workflows/ci-rust.yml

```diff
@@ -25,9 +25,9 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v4
 
-      - uses: prefix-dev/setup-pixi@v0.8.11
+      - uses: prefix-dev/setup-pixi@v0.8.14
```

Comment:
Interesting that `dependabot` didn't do that. Maybe it's not the day of the week yet.

Reply:
Actually I had the same thought and went back to check. Last time `dependabot` ran was 2 weeks ago.... And the new version of setup prefix came out 5 days ago. Something weird is going on for sure